tests/test_transform.py

```diff
@@ -180,10 +180,94 @@ def checkTransform(self, fromName, toName):
                 with self.assertRaises(InvalidParameterError, msg=msg):
                     transformClass(badPolyMap)
 
+    def checkGetInverse(self, fromName, toName):
+        """Test Transform<fromName>To<toName>.getInverse
+
+        Parameters
+        ----------
+        fromName, toName : string
+            the prefixes of the transform's endpoints (e.g., "Point2" for a
+            Point2Endpoint)
+        """
+        transformClassName = "Transform{}To{}".format(fromName, toName)
+        transformClass = getattr(afwGeom, transformClassName)
+        baseMsg = "transformClass={}".format(transformClass.__name__)
+        for nIn in self.goodNaxes[fromName]:
+            for nOut in self.goodNaxes[toName]:
+                # Transform with both forward and inverse transforms
+                self.checkGetInverseWithMap(
+                    transformClass(makePolyMap(nIn, nOut)),
+                    nIn,
+                    "{}, Map={}, nIn={}, nOut={}".format(
+                        baseMsg, "Invertible", nIn, nOut))
+                # TODO: add more test cases once DM-9826 resolved
+
+    def checkGetInverseWithMap(self, transform, nIn, msg):
+        """Test Transform<fromName>To<toName>.getInverse with a specific mapping.
```

Comment:
What do you mean by "with a specific mapping"? A transform can be constructed with a mapping or a frame set, but the results is that it contains a frame set in either case.

To my mind the name suggests that the user will provide a mapping as an argument to this method (I am not suggesting you do that; the current test seems excellent).

Can you call this `checkGetInverse` and skip the `WithMap`?